using_openshift/executing_remote_commands.adoc

```diff
@@ -0,0 +1,76 @@
+= Executing Remote Commands
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+OpenShift takes advantage of a feature built into Kubernetes to support
+executing commands in containers. This is implemented using HTTP along with a
+multiplexed streaming protocol such as link:http://www.chromium.org/spdy[*SPDY*]
+or link:https://http2.github.io/[*HTTP/2*], and can be used to connect to a
+container via SSH as well.
+
+== Client Operation
+Support for remote container command execution is
+link:../using_openshift/cli.html#common-cli-operations[built into the CLI]:
+
+****
+`$ osc exec -p _<pod>_ [-c _<container>_] -- _<command>_ _[<arg 1> ... <arg n>]_`
+****
+
+Clients initiate the execution of a remote command in a container by issuing a
+request to the Kubernetes API server:
+
+[options="nowrap"]
+----
+/proxy/minions/node123.openshift.com/exec/myns/mypod/mycontainer?command=date
```

Comment:
I'm not sure about Origin right now, but I know that for Kubernetes upstream, `minions` is now `nodes` in the URL in order for this to work, at least as of v1beta3.

@smarterclayton what is appropriate here today?